bottlecap/src/traces/stats_aggregator.rs

```diff
@@ -169,7 +175,7 @@ mod tests {
             &HashMap::new(),
         ));
         let (_, concentrator) = StatsConcentratorService::new(config, tags_provider);
-        let mut aggregator = StatsAggregator::new(640, concentrator);
+        let mut aggregator = StatsAggregator::new(720, concentrator);
```

Comment:
@duncanista I'm trying to understand this test in order to fix it. Why "The batch should only contain the first 2 payloads" when "Payload below is 115 bytes" and aggregator's `max_content_size_bytes` is 640 bytes?